auth-api/src/auth_api/models/documents.py

```diff
@@ -0,0 +1,39 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Table for storing all static documents.
+
+Documents which are static in nature are stored in this table  ie.terms of use
+"""
+
+from sqlalchemy import Column, Integer, String, Text, desc
+
+from .base_model import BaseModel
+from .db import db
+
+
+class Documents(BaseModel):
+    """This is the model for a documents."""
+
+    __tablename__ = 'documents'
+
+    # TODO version concept is not well refined..this is the first version..refine it
+    version_id = Column(Integer, primary_key=True, autoincrement=False)
+    type = Column('type', String(20), nullable=False)
+    content = Column('content', Text)
```

Comment:
Would it be better if we add start time and expiry time to the model, so that we can set the terms of service upfront with start date. The query would get whatever is latest for the current date.